articles/idam/idam.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ID & Access Management API | UKCloud Ltd
+description: Provides information on managing permissions using the ID & Access Management API
+services: idam
+author: Simon Fish
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: IDAM
+toc_fullpath: Getting Started/idam.md
+toc_mdlink: idam.md
+---
+
+# Identity and Access Management (IDAM)
+
+UKCloud's **ID** & **A**ccess **M**anagement API (IDAM) allows users to manage user access to products within their account. Adding a user to a group changes which permissions they have.
+
+Full documentation for the IDAM API is available on [SwaggerHub](https://app.swaggerhub.com/apis/ukcloud/idam-service).
+
+## Typical usage
+
+In order to add a user to a permission group, you can list the users and permission groups you have access to in order to find the IDs for both.
```

Comment:
Change to:
To add a user to a permission group, you first need to list the users and permission groups to find the IDs for both. Then you can use those IDs to add the user to the appropriate group.